tests/plugins/org.polarsys.capella.test.validation.rules.ju/src/org/polarsys/capella/test/validation/rules/ju/testcases/i/Rule_I_47_IncorrectImagePath.java

```diff
@@ -75,6 +86,22 @@ public void test() throws Exception {
     assertEquals("Bad diagnostic children number", 1, children.size());
     assertEquals("Bad diagnostic severity", Diagnostic.ERROR, children.get(0).getSeverity());
     assertTrue("Bad diagnotic message", children.get(0).getMessage().contains("(Image) The absolute path"));
+    List<IMarker> markers = new ArrayList<IMarker>();
```

Comment:
Move this up to one of the abstract classes, like AbstractRulesOnIntegrityTest ?